internals/cli/cmd_enter.go

```diff
@@ -29,7 +42,14 @@ These subcommands are currently supported:
 (1) Services are not started.
 (2) No logs on stdout unless -v is used.
 (3) Services continue running after the subcommand succeeds.
-`
```

Comment:
Can we please preserve the existing convention of declaring the short and long help texts at the top of the file using variables?   Not only this makes it easy to have a glimpse of what the command does upfront, but it avoids what is happening here, which is having a type construction being broken down in the middle in an ugly way, making it harder to read both the help and the code. This was the original reason for this convention, and it still rings true seeing the code now.